editor/src/components/canvas/ui-jsx-canvas-renderer/remix-scene-component.tsx

```diff
@@ -24,6 +24,9 @@ export const RemixSceneComponent = React.memo((props: React.PropsWithChildren<Re
   const { style, getLoadContext, startingRoute, ...remainingProps } = props
 
   const sceneStyle: React.CSSProperties = {
+    // eslint-disable-next-line
+    // @ts-ignore
+    container: `${SceneContainerName} / inline-size`,
```

Comment:
why is the `@ts-ignore` needed here? If it's because of the `React.CSSProperties` type, I'd rather extend the type:
```typescript
type CSSPropertiesWithContainer = React.CSSProperties & { container: string }
```
and use that instead of `React.CSSProperties`

Reply:
Yeah, this please

Reply:
yeah, React 19 already supports it, but it's a good idea to monkey patch it for now.
probably 
```
type CSSPropertiesWithContainer = { container: string } & React.CSSProperties
```
to not override if we'll upgrade and forget to change it back.
will do!

Reply:
ok, after some digging, turns out we had `csstype` locked to `3.0.3`. simply [upgrading](https://github.com/concrete-utopia/utopia/pull/6684/commits/7b6478c158512768581676b9489c675727d53b56) it to `3.1.3` did the trick, without the need for the React upgrade (that uses it).